sympy/algebras/quaternion.py

```diff
@@ -720,3 +720,422 @@ def to_rotation_matrix(self, v=None):
+        >>> q1 = Quaternion(1, 4, 4, 4)
+        >>> q2 = Quaternion(2, 8, 8, 8)
+        >>> Quaternion.coplanar(q1, q2)
+        True
+
+        >>> q1 = Quaternion(2, 8, 13, 12)
+        >>> Quaternion.coplanar(q1, q2)
+        False
+
+        See Also
+        ========
+
+        ternary_coplanar
+        is_pure
+
+        """
+        if (q1.is_zero_quaternion()) or (q2.is_zero_quaternion()):
+            raise ValueError('Any of the provided quaternions must not be 0')
+
+        aq = q1.axis()
+        ar = q2.axis()
+
+        return fuzzy_or([(aq - ar).is_zero_quaternion(), (aq + ar).is_zero_quaternion()])
+
+    @classmethod
+    def ternary_coplanar(cls, q1, q2, q3):
+        """
+        Returns True if the axis of the pure quaternions seen as 3D vectors
+        q1, q2, and q3 are coplanar else returns False.
+
+        Explanation
+        ===========
+
+        Two general quaternions are coplanar (in this arc sense) when their axes are parallel.
+
+        Parameters
+        ==========
+
+        q1 : a pure Quaternion.
+        q2 : a pure Quaternion.
+        q3 : a pure Quaternion.
+
+        Examples
+        ========
+
+        >>> from sympy.algebras.quaternion import Quaternion
+        >>> q1 = Quaternion(0, 4, 4, 4)
+        >>> q2 = Quaternion(0, 8, 8, 8)
+        >>> q3 = Quaternion(0, 24, 24, 24)
+        >>> Quaternion.ternary_coplanar(q1, q2, q3)
+        True
+
+        >>> q1 = Quaternion(0, 8, 16, 8)
+        >>> q2 = Quaternion(0, 8, 3, 12)
+        >>> Quaternion.ternary_coplanar(q1, q2, q3)
+        False
+
+        See Also
+        ========
+
+        axis
+        is_pure
+
+        """
+
+        if fuzzy_not(q1.is_pure()) or fuzzy_not(q2.is_pure()) or fuzzy_not(q3.is_pure()):
+            raise ValueError('The provided quaternions must be pure')
+
+        M = Matrix([[q1.b, q1.c, q1.d], [q2.b, q2.c, q2.d], [q3.b, q3.c, q3.d]]).det()
+        return M.is_zero
+
+    def parallel(self, other):
+        """
+        Returns True if the two pure quaternions seen as 3D vectors are parallel else returns False.
+
+        Explanation
+        ===========
+
+        Two pure quaternions are called parallel when their product is commutative.
+
+        Parameters
+        ==========
+
+        other : a Quaternion
+
+        Examples
+        ========
+
+        >>> from sympy.algebras.quaternion import Quaternion
+        >>> q = Quaternion(0, 4, 4, 4)
+        >>> q1 = Quaternion(0, 8, 8, 8)
+        >>> q.parallel(q1)
+        True
+
+        >>> q1 = Quaternion(0, 8, 13, 12)
+        >>> q.parallel(q1)
+        False
+
+        """
+
+        if fuzzy_not(self.is_pure()) or fuzzy_not(other.is_pure()):
+            raise ValueError('The provided quaternions must be pure')
+
+        return (self*other - other*self).is_zero_quaternion()
+
+    def orthogonal(self, other):
+        """
+        Returns True if the two pure quaternions seen as 3D vectors are orthogonal else returns False.
```

Comment:
Maybe it would be better to say something like:
```
True if the vector parts of ``q1`` and ``q2`` are parallel.
```
Here (as in many other places) the return value can be True, False or None but this explicitly says that it is either True or False which is incorrect.

I think that a better way to clarify this is to say "True if..." as I suggest above and then to have a Returns section that clarifies that the return value uses three-valued logic and can be True, False or None. That way the initial line of the docstring is short and clear but the full detail is still provided.